examples/cs.py

```diff
@@ -6,6 +6,7 @@
 from cstool.phonon import phonon_cs_fn
 from cstool.inelastic import inelastic_cs_fn
 from cstool.compile import compute_tcs_icdf
+from cstool.compile import compute_tcs
```

Comment:
These two lines can be merged in one. Would make it more consistent with the other imports.